sdk/src/main/java/com/kin/ecosystem/history/view/OrderHistoryRecyclerAdapter.java

```diff
@@ -65,6 +65,12 @@ private void initSizes(Context context) {
         }
     }
 
+    private void initStrings(Context context) {
+        if(TRANSACTION_FAILED_MSG == null) {
+            TRANSACTION_FAILED_MSG = context.getString(R.string.kinecosystem_transaction_failed);
```

Comment:
I don't like TRANSACTION_FAILED_MSG being null. I understand that init(context) must be called first otherwise we don't have context but having null open a window (very small one I admit) for NPE

Reply:
i'll set empty value